plugins/techdocs-backend/api-report.md

```diff
@@ -16,6 +19,49 @@ import { PublisherBase } from '@backstage/techdocs-common';
 // @public (undocumented)
 export function createRouter({ preparers, generators, publisher, config, logger, discovery, }: RouterOptions): Promise<express.Router>;
 
+// @public (undocumented)
+export class DefaultTechDocsCollator implements DocumentCollator {
+    constructor({ discovery, locationTemplate, logger, catalogClient, parallelismLimit, }: {
+        discovery: PluginEndpointDiscovery;
+        logger: Logger_2;
+        locationTemplate?: string;
+        catalogClient?: CatalogApi;
+        parallelismLimit?: number;
+    });
+    // (undocumented)
+    protected applyArgsToFormat(format: string, args: Record<string, string>): string;
+    // (undocumented)
+    protected discovery: PluginEndpointDiscovery;
+    // (undocumented)
+    execute(): Promise<TechDocsDocument[]>;
+    // (undocumented)
+    protected locationTemplate: string;
+    // (undocumented)
+    readonly type: string;
+}
+
+// @public (undocumented)
+export interface MkSearchIndexDoc {
+    // (undocumented)
+    location: string;
+    // (undocumented)
+    text: string;
+    // (undocumented)
+    title: string;
+}
```

Comment:
I think this shouldn't be exposed as a public interface. Shouldn't be necessary, right?